pkg/webhook/validation/dynakube/dynakube_name_test.go

```diff
@@ -24,3 +25,29 @@ func TestNameStartsWithDigit(t *testing.T) {
 		})
 	})
 }
+
+func TestNameTooLong(t *testing.T) {
+	t.Run(`normal name`, func(t *testing.T) {
+		assertAllowedResponse(t, &dynatracev1beta1.DynaKube{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "dynakube",
+			},
+			Spec: dynatracev1beta1.DynaKubeSpec{
+				APIURL: "https://tenantid.doma.in/api",
+			},
+		})
+	})
+	t.Run(`name too long`, func(t *testing.T) {
+		n := dynatracev1beta1.MaxNameLength + 2
+		letters := make([]rune, n)
+		for i := 0; i < n; i++ {
+			letters[i] = 'a'
+		}
+		errorMessage := fmt.Sprintf(errorNameTooLong, dynatracev1beta1.MaxNameLength)
+		assertDeniedResponse(t, []string{errorMessage}, &dynatracev1beta1.DynaKube{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: string(letters),
```

Comment:
see you need a string here :) so strings.Repeat("A", dynatracev1beta1.MaxNameLength + 2)

Reply:
https://pkg.go.dev/strings#Repeat

Reply:
❤️ done